README.md

```diff
@@ -343,3 +343,12 @@ rpds = { version = "<version>", features = ["serde"] }
 ```
 
 <!-- cargo-rdme end -->
+
+### Bindings
```

Comment:
I excluded the section from cargo (assuming that's what cargo-rdme does), under the assumption Rust people don't care about this section, but let me know if you want it above the line!

Reply:
I think it is a good idea to add to the crate's doc.  Can you add this to the `lib.rs` rustdoc (and the run `cargo rdme` or sync the README.md manually?